app/frontend/src/components/forms/ExportSubmissions.vue

```diff
@@ -35,11 +34,15 @@ export default {
           'End date should be greater than start date.',
       ],
       exportFormat: 'json',
-      inputFilter: '',
+      formFieldsSearchFilter: '',
+      formVersions: [],
+      // This has the form fields in the table
+      // Table selection does not work if we use a computed value
       items: [],
-      selected: [],
-      showFieldsOptions: false,
-      startDate: moment(Date()).format('YYYY-MM-DD'),
+      loading: false,
```

Comment:
added a loading variable to check if we've initiated a fetch for form fields. could not just simply check if we retrieved any data yet because sometimes the data returned is just empty.